shared-libs/logger/src/index.js

```diff
@@ -0,0 +1,82 @@
+if (!process || process.browser) {
+  module.exports = console;
+} else {
```

Comment:
Having different `module.exports` code depending on this branch makes it really hard to work out what it's going to do, especially if you're calling it from another shared lib and you don't know what context you're in. I think it'd be easier to read if it was something like...

```js
const create() => {
  if (!process || process.browser) {
    return console;
  } else {
    return require('./node-logger.js').create();
  }
};

module.exports = create();
module.exports.DATE_FORMAT = DATE_FORMAT;
```